### PR TITLE
Added Aina Cook

### PR DESCRIPTION
**Pull request by Tsikitsiky**
Hey, 
This is my new PR, please have a look.

Thank you, 
Sarah

### CODE REVIEW

File: a/aina-cook.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Aina cook
```
Comment (ndorfin) [inline a/aina-cook.md]: Her family name starts with a capital C

File: a/aina-cook.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+![Aina cook](aina-cook.jpg)
```
Comment (ndorfin) [inline a/aina-cook.md]: Capital C in the alt-text please.

File: a/aina-cook.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+Aina cook in her real name Aina Elison Cook is a biracial Malagasy Americain singer whose hometown is Antananarivo capital of Madagascar and currently lives in St Louis, MO. She is the singer/writer of her songs.
```
Comment (ndorfin) [inline a/aina-cook.md]: And here too, capital C for Cook.

**Review by ndorfin — CHANGES_REQUESTED** [review]

**Review by ndorfin — APPROVED** [review]
Much better, thank you!

File: a/aina-cook.md (hunk 3 above)
Comment (Tsikitsiky) [reply a/aina-cook.md]: I think I've the changes, please check.

**Pull request merged**